dora/core/common/src/main/java/alluxio/MembershipType.java

```diff
@@ -0,0 +1,10 @@
+package alluxio;
+
+/**
+ * MembershipManager type
+ */
+public enum MembershipType {
+  STATIC,
+  ETCD,
+  NONE
```

Comment:
based on other part of the code, maybe call this NOOP?
```suggestion
  NOOP
```

Reply:
done